backend/kernelspace/proxier_events.go

```diff
@@ -172,24 +174,26 @@ func (proxier *Proxier) newEndpointInfo(baseInfo *BaseEndpointInfo, _ *ServicePo
 func (proxier *Proxier) newServiceInfo(port *localv1.PortMapping, service *localv1.Service, baseInfo *BaseServiceInfo) ServicePort {
 	info := &serviceInfo{BaseServiceInfo: baseInfo}
 	preserveDIP := service.Annotations["preserve-destination"] == "true"
-	//localTrafficDSR := service.ExternalTrafficPolicy == v1.ServiceExternalTrafficPolicyTypeLocal
-	//err := hcn.DSRSupported()
-	//if err != nil {
-	//    preserveDIP = false
-	//    localTrafficDSR := false
-	//}
-	localTrafficDSR := false
+	localTrafficDSR := service.ExternalTrafficToLocal      // TODO: verify this is the correct field in the KPNG localv1 API
```

Comment:
@jayunit100 or @knabben - Is am I doing the correct assignments for `localTrafficDSR` and `internalTrafficLocal` ?
These wre previously checking against core.v1 types

Reply:
I think it's correct, the upstream model is different from the kpng one related to these bool services parameters